### ISSUE TITLE
[suggestion] Downgrade button for Webui/Disable dev channel for stable

### ISSUE THREAD

**Issue by bigretromike** [ShokoAnime/Shoko-WebUI#967]
In my previous issues you guys were saying I was using "old" webui even if it came with stable installer.
I updated it with 'dev' channel. 

4.2.2 is non-compatible with that version, I'm even unable to switch to 'stable' because of many errors, missing things etc. 
A simple "revert to stable" button next/under the 'refresh' version when it detects 'dev' in version string. Would be nice. 

Just saying or maybe detect server version and if its stable (0? on 4 place) hide dev channel from update ?

**Comment by harshithmohan** [comment]
I failed to explain properly. We are not looking into issues for the stable webui because every part of it has been replaced in v2. There is no point in fixing anything in it and we are close to a new stable release.

The daily webui does have a check and redirects to unsupported page if you are stable server, or if an error occurs, it goes to the error page where you have an option to force "update" to stable webui.

> Just saying or maybe detect server version and if its stable (0? on 4 place) hide dev channel from update ?
Daily webui might work with the new stable when we get it. So we now check using the server release date/version. We will stop an update to dev webui if you are on stable server.
